app/routers/items.py

```diff
@@ -30,10 +31,10 @@ def create_item(db: Session = Depends(get_db), *, item: NewItem) -> None:
 
 @router.get("/items")
 def get_items(
-    db: Session = Depends(get_db), *, only_active: bool = True
+    db: Session = Depends(get_db), *, only_uncomplete_items: bool = True
 ) -> List[PydanticItem]:
-    if only_active is True:
-        items = db.query(Item).filter(Item.status == 1)
+    if only_uncomplete_items is True:
```

Comment:
Should be "only_uncompleted_items"...missing a "d"

Reply:
Resolved via [`00bca94` (#56)](https://github.com/iancleary/ivy-lee-backend/pull/56/commits/00bca949379e1757174a0cd358a801552146780b)